canonical-websites/www.ubuntu.com
# Fix markup errors in site footer and UA service description.

## Done

Fixed markup errors in the markup 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/ubuntu-advantage/service-description](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description)
- Check that the nav links to the appendices work.
